CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # v4.3.2
-* Added: The ability to perform a term query by prefix using a new Compare.STARTS_WITH predicate. 
+* Added: The ability to perform a term query by prefix using a new Compare.STARTS_WITH predicate.
+* Changed: Accumulo to lazily load metadata values 
 * Fixed: AccumuloGraphLogger was throwing an error when trying to log a field from a Scanner that was removed in Accumulo 1.8.0
```

Comment:
Should you mention that you converted Metadata to an Interface? Code will need to be updated to `Metadata.create` instead of `new Metadata`.